### ISSUE TITLE
[VersionControl] Error opening Checkout Dialog

### ISSUE THREAD

**Issue by jsuarezruiz** [mono/monodevelop#7108]
Cannot open the Checkout Dialog without an opened solution.

```
ERROR [2019-02-11 08:21:31Z]: Error while executing command: Checkout...
System.NullReferenceException: Object reference not set to an instance of an object
at MonoDevelop.Ide.MessageService.RunCustomDialog (MonoDevelop.Components.Dialog dlg, MonoDevelop.Components.Window parent) [0x0007c] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/MessageService.cs:337 
at MonoDevelop.Ide.MessageService.RunCustomDialog (MonoDevelop.Components.Dialog dialog) [0x00001] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/MessageService.cs:314 
at MonoDevelop.VersionControl.CheckoutCommand.Run () [0x00009] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/addins/VersionControl/MonoDevelop.VersionControl/MonoDevelop.VersionControl/CheckoutCommand.cs:21 
at MonoDevelop.Components.Commands.CommandHandler.Run (System.Object dataItem) [0x00001] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandHandler.cs:75 
at MonoDevelop.Components.Commands.CommandHandler.InternalRun (System.Object dataItem) [0x00001] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandHandler.cs:39 
at MonoDevelop.Components.Commands.CommandManager.DefaultDispatchCommand (MonoDevelop.Components.Commands.ActionCommand cmd, MonoDevelop.Components.Commands.CommandInfo info, System.Object dataItem, System.Object target, MonoDevelop.Components.Commands.CommandSource source) [0x00096] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs:1584 
at MonoDevelop.Components.Commands.CommandManager.DispatchCommand (System.Object commandId, System.Object dataItem, System.Object initialTarget, MonoDevelop.Components.Commands.CommandSource source, System.Nullable`1[T] time, MonoDevelop.Components.Commands.CommandInfo sourceUpdateInfo) [0x00455] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs:1547`
`ERROR [2019-02-11 08:21:33Z]: Error while executing command: Checkout...`
`System.NullReferenceException: Object reference not set to an instance of an object`
`at MonoDevelop.Ide.MessageService.RunCustomDialog (MonoDevelop.Components.Dialog dlg, MonoDevelop.Components.Window parent) [0x0007c] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/MessageService.cs:337`
`at MonoDevelop.Ide.MessageService.RunCustomDialog (MonoDevelop.Components.Dialog dialog) [0x00001] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide/MessageService.cs:314`
`at MonoDevelop.VersionControl.CheckoutCommand.Run () [0x00009] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/addins/VersionControl/MonoDevelop.VersionControl/MonoDevelop.VersionControl/CheckoutCommand.cs:21`
`at MonoDevelop.Components.Commands.CommandHandler.Run (System.Object dataItem) [0x00001] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandHandler.cs:75`
`at MonoDevelop.Components.Commands.CommandHandler.InternalRun (System.Object dataItem) [0x00001] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandHandler.cs:39`
`at MonoDevelop.Components.Commands.CommandManager.DefaultDispatchCommand (MonoDevelop.Components.Commands.ActionCommand cmd, MonoDevelop.Components.Commands.CommandInfo info, System.Object dataItem, System.Object target, MonoDevelop.Components.Commands.CommandSource source) [0x00096] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs:1584`
`at MonoDevelop.Components.Commands.CommandManager.DispatchCommand (System.Object commandId, System.Object dataItem, System.Object initialTarget, MonoDevelop.Components.Commands.CommandSource source, System.Nullable`1[T] time, MonoDevelop.Components.Commands.CommandInfo sourceUpdateInfo) [0x00455] in /Users/jsuarezruiz/Projects/MonoDevelop/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Components.Commands/CommandManager.cs:1547
```

> VS bug [#789305](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/789305)

**Comment by jsuarezruiz** [comment]
Sent related PR: https://github.com/mono/monodevelop/pull/7127

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.0.0.1526 (master)

Pull Request #7127 merged by: Lluis Sanchez
Author: mono
Commit: faf751426fc8f10fffe31ba650fffe01adb9422d (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.0.0.2306 (release-8.0)

Pull Request #7127 merged by: Lluis Sanchez
Author: mono
Commit: faf751426fc8f10fffe31ba650fffe01adb9422d (xamarin/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #7127 merged by: Lluis Sanchez
Author: mono
Commit: faf751426fc8f10fffe31ba650fffe01adb9422d (xamarin/monodevelop)
